arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -292,54 +388,138 @@
       "type": "object",
       "properties": {
         "storageAccount": {
-          "$ref": "#/definitions/StorageAccountProperties"
+          "$ref": "#/definitions/StorageAccountBaseProperties"
         },
         "loginServer": {
           "type": "string"
         },
         "username": {
           "type": "string"
         },
-        "key": {
-          "type": "string"
-        },
         "creationDate": {
           "format": "date-time",
           "type": "string"
         }
       }

```

Comment:
Please add descriptions
RegistryProperties: "The properties for a container registry."
StorageAccountBaseProperties: "The base properties for a storage account."
RegistryCreateParameters: "The parameters for creating a container registry."
RegistryPropertiesCreateParameters: "The properties for creating a container registry."
StorageAccountProperties: "The properties for a storage account."
RegistryUpdateParameters: "The parameters for updating a container registry."
ResourceList[Registry]: "The list of container registries."
SubscriptionNotification: ?
SubscriptionProperties: ?